### PR TITLE
Fixed missing arguments in `lr_find` call

### LINKED ISSUE #6783: Trainer tuning fails when used with a LightningDataModule
The `lr_find` call must take in `train_dataloader`, `val_dataloaders` and `datamodule` in order to support dataloaders separately.

https://github.com/PyTorchLightning/pytorch-lightning/blob/3e3175d0748ef461bf13b2258c8acdbe3a7bc8a5/pytorch_lightning/tuner/tuning.py#L64

### PR DESCRIPTION
**Pull request by THasthika**
There seem to be 3 arguments missing in the `lr_find` call in the tunining.py file.

## What does this PR do?

Adds the 3 arguments stated in the issue to the `lr_find` call.

Fixes #6783 

## Before submitting
- [ ] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [ ] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [ ] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [ ] Did you make sure to update the documentation with your changes? (if necessary)
- [ ] Did you write any new necessary tests? (not for typos and docs)
- [ ] Did you verify new and existing tests pass locally with your changes?
- [ ] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Review by Borda — APPROVED** [review]

**Review by SkafteNicki — APPROVED** [review]

**Review by awaelchli — APPROVED** [review]

**Comment by ananthsub** [comment]
are there tests for this?

**Review by Borda — APPROVED** [review]
